Trappist/src/Promact.Trappist.DomainModel/Models/Question/Question.cs

```diff
@@ -27,5 +27,8 @@ public class Question:BaseModel
         public virtual SingleMultipleAnswerQuestion SingleMultipleAnswerQuestion { get; set; }
 
         public virtual CodeSnippetQuestion CodeSnippetQuestion { get; set; }
+
+        [ForeignKey("CreatedBy")]
+        public virtual ApplicationUser ApplicationUser { get; set; }
```

Comment:
Cannot set two foreign key of `ApplicationUser` table in `Question` table. So I have to manually insert `ApplicationUserId` into `UpdatedBy` field. But there will be a problem. While updating the question the `CreatedBy` field will get overwrite with the `ApplicationUserId` of the user updating the question along with the `UpdatedBy` field (ref. Line 60) which should not happen.
**Possible solution is  to create relationship with `UpdatedBy` field only and insert `CreatedBy` manually.** Example Code:-
`question.ApplicationUser = await _userManager.FindByEmailAsync(userEmail);`
`question.CreatedBy = question.ApplicationUser.Id;`